src/main/java/com/kainos/ea/nameConfiguration.java

```diff
@@ -1,8 +1,11 @@
 package com.kainos.ea;
 
 import io.dropwizard.Configuration;
+
```

Comment:
Why these additions? It highlighted other import *'s lol

Reply:
Not sure why they were present, but we've since removed them!